### ISSUE TITLE
Straddle board disappearing

### ISSUE THREAD

**Issue by Toootla** [Alex-the-666/AlexsMobs#1849]
When breaking the board enchanted with returning board, the item doesn't appear in your inventory, even with plenty of open slots, or drop as an item.

Also when leaving and joining the world while standing on the board, it vanishes.

Minecraft Version: 1.20.1
Mod Version: alexsmobs-1.22.6.jar

Tested both on a server and singleplayer

**Comment by ChicknTurtle** [comment]
Happened to my friend to, they broke it and couldn't find it anywhere.

**Comment by Johncombat** [comment]
Yeah just happened to me as well, first encountered it in a server and tested to make sure in singleplayer.

**Comment by maxrr** [comment]
Still facing this issue as well. Playing Better MC forge version 26. No error messages appear in server console.

**Comment by adrionl** [comment]
still hasnt been fixed bummer
